app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(
+        data: list,
+        wife_key: str = "wife",
+        husband_key: str = "husband"
+) -> list:
+    person_list = [Person(person_data["name"],
+                   person_data["age"]) for person_data in data]
+
+    for person_data in data:
+        person = Person.people[person_data["name"]]
+
+        if person_data.get(wife_key) is not None:
+            wife_id = person_data.get(wife_key)
+            person.wife = Person.people.get(wife_id, None)
```

Comment:
`None` is a default return value in `.get()`, there is no need to indicate this, so remove `None` from `.get()`, please.